### PR TITLE
wallet: token value drawer

### LINKED ISSUE #17711: Wallet: token value drawer
Implement Token Value list item drawer UI.

Designs: https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=4695-494443&mode=design&t=qkjT8nW1A4bET2x0-4

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/17711

This PR implements token value drawer on home screen and account screen.

[Designs](https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=4695-494198&mode=design&t=Ck8GY17FT3H9tUEt-0)

Demo:

https://github.com/status-im/status-mobile/assets/29354102/1b4bc31a-1456-4bc5-ae07-b4ee64a3024d

### CODE REVIEW

File: src/quo/components/inputs/input/view.cljs [hunk 1]
```diff
@@ -150,7 +150,7 @@
               :auto-complete       :password
               :secure-text-entry   (not @password-shown?)
               :right-icon          {:style-fn  style/password-icon
-                                    :icon-name (if @password-shown? :i/hide :i/reveal)
+                                    :icon-name (if @password-shown? :i/hide-password :i/reveal)
```
Comment (J-Son89) [inline src/quo/components/inputs/input/view.cljs:153]: just wondering if you double checked this with design? seems a bit weird for them to update the icon in one place (I more so mean the reference key to the icon). Did they update other uses of this across the designs?

File: src/status_im2/contexts/wallet/common/token_value/view.cljs [hunk 2]
```diff
@@ -0,0 +1,40 @@
+(ns status-im2.contexts.wallet.common.token-value.view
```
Comment (J-Son89) [inline src/status_im2/contexts/wallet/common/token_value/view.cljs:1]: Nice! :D

**Review by mmilad75 — APPROVED** [review]

File: src/quo/components/inputs/input/view.cljs (hunk 1 above)
Comment (OmarBasem) [reply src/quo/components/inputs/input/view.cljs:153]: @J-Son89 What do you mean in one place? On the icon resources page, there is a comment on the icon that they updated icon's name (`hide` to `hide-password`). 

https://www.figma.com/file/qLLuMLfpGxK9OfpIavwsmK/Iconset?type=design&node-id=3239-987&mode=design&t=MB5DkyAqd4hPgY8o-0

Comment (J-Son89) [reply src/quo/components/inputs/input/view.cljs:153]: perfect, sounds good 👍

**Review by Francesca-G — APPROVED** [review]
Nice job ✨

**Pull request merged**